### PR TITLE
add datahub upgrade runbook

### PR DESCRIPTION
**Pull request by tom-webber**
Runbook for DataHub upgrades
- Add docs for upgrading DataHub

Minor updates for 'Joining Our Team'
- update slack channel details
- add link for ceremonies

documents ministryofjustice/data-catalogue#25

### CODE REVIEW

File: source/documentation/runbooks/005-upgrade-datahub.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+1. Check the DataHub helm releases to
+[check for the latest release that's compatible with the destination version](https://github.com/acryldata/datahub-helm/blame/master/charts/datahub/Chart.yaml#L10)
+1. Check the [DataHub helm release changes from current deployed version to future version](https://github.com/acryldata/datahub-helm/compare/datahub-0.3.30...datahub-0.4.6)
+for any changes that need to be propagated to the helm chart values files
+(make sure to set the 'base' and 'compare' to the correct release values in this view)
+1. Submit the upgrade release PR and send an 'alert' message to the [#data-catalogue-dev](https://moj.enterprise.slack.com/archives/C06UQ5HSR25) channel to let others know that DataHub may be disrupted
+1. Deploy to dev and test out the [DataHub interface](https://datahub-catalogue-dev.apps.live.cloud-platform.service.justice.gov.uk/),
+ensure that [Find MoJ Data dev](https://data-platform-find-moj-data-dev.apps.live.cloud-platform.service.justice.gov.uk/)
+continues to function ([run a test deployment](https://github.com/ministryofjustice/find-moj-data/actions/workflows/test-and-deploy.yml))
```
Comment (murdo-moj) [inline source/documentation/runbooks/005-upgrade-datahub.html.md.erb:25]: Do we want to mention about how access tokens can get reset when updating datahub, so they need to be refreshed in `find-moj-data`?

File: source/documentation/runbooks/005-upgrade-datahub.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# <%= current_page.data.title %>
+
+There is no prescribed cadence for updates to DataHub, but new minor versions bumps are typically released every couple of months.
+Due to the rapid pace of feature development within the platform
+we aim to be close to the most recent release (without unnecessary exposure to unintended breaking changes).
+
+The upgrade process for DataHub is relatively straightforward as it is handled by a [dedicated upgrade job](https://github.com/acryldata/datahub-helm/tree/master/charts/datahub/templates/datahub-upgrade):
+
+1. Check the [DataHub 'Updating DataHub' page for breaking changes](https://datahubproject.io/docs/how/updating-datahub#breaking-changes)
+1. Check the DataHub helm releases to
```
[the remark this answers is not in the document]
Comment (tom-webber) [reply source/documentation/runbooks/005-upgrade-datahub.html.md.erb:17]: They increase sequentially in the rendered markdown. I've left them as 'numeric placeholders' so it's easier and less fiddly to edit the actual markdown code

File: source/documentation/runbooks/005-upgrade-datahub.html.md.erb (hunk 1 above)
Comment (tom-webber) [reply source/documentation/runbooks/005-upgrade-datahub.html.md.erb:25]: I'm waiting on Mitch to get back to us on whether that is a mandatory step of the upgrade. I've added him as a reviewer here, but I'll also ping him on Slack

Comment (mitchdawson1982) [reply source/documentation/runbooks/005-upgrade-datahub.html.md.erb:25]: As we have found that at least in the most recent upgrade, pre-existing PAT's continue to work as expected, I feel the content we already have is accurate.

**Pull request merged**